components/expand-collapse/demo/expand-collapse.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta http-equiv="X-UA-Compatible" content="ie=edge">
+	<meta charset="UTF-8">
+	<link rel="stylesheet" href="../../demo/styles.css" type="text/css">
+	<script src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+	<script type="module">
+		import '../../button/button.js';
+		import '../../demo/demo-page.js';
+		import '../expand-collapse.js';
+	</script>
+</head>
+
+<body unresolved>
+
+	<d2l-demo-page page-title="d2l-expand-collapse">
+
+		<h2>Default</h2>
+		<d2l-demo-snippet>
+			<template>
+				<d2l-button primary>Toggle</d2l-button>
+				<d2l-expand-collapse>
```

Comment:
Another scenario worth checking is with `expanded` initially `true`.  Be careful that any delay caused by the snippet template being stamped and inserted isn't misleading. If in doubt, try it outside a snippet.  I'd expect not to see an animation in such case.

Reply:
I think it would be worthwhile adding a second snippet that has `expanded` initially set.